app/experimenter/visualization/api/v1/views.py

```diff
@@ -0,0 +1,40 @@
+import json
+
+from django.core.files.storage import default_storage
+from django.http import HttpResponse
+from django.views import View
+
+from experimenter.experiments.models import Experiment
+
+
+class VisualizationView(View):
+    def load_data_from_gcs(self, filename):
+        return (
+            json.loads(default_storage.open(filename).read().decode("utf8"))
+            if default_storage.exists(filename)
+            else []
+        )
+
+    def get(self, request, *args, **kwargs):
+        experiment_json = {}
+
+        slug = self.kwargs["slug"]
+        current_expermient_arr = Experiment.objects.get_prefetched().filter(slug=slug)
+        if len(current_expermient_arr) < 1:
+            return HttpResponse(json.dumps(experiment_json))
+
+        recipe_slug = current_expermient_arr[0].recipe_slug.replace("-", "_")
```

Comment:
Oh I just remembered on `NimbusExperiment` this is just `slug`, not `recipe_slug`